Utt_Christelle_Homework_Software_Engineering_1.py

```diff
@@ -0,0 +1,138 @@
+"""
+TASK 1
```

Comment:
✅ Well done on completing the Task 1. 

Points 6/8
-1 When applying discount the total_price should be changed to reflect the discount
-0.5 add_item method violates the SRP principle
-0.5 reset method should reset all attributes including the total_price and discount